ArcGISRuntimeSDKQt_CppSamples/Maps/ChangeBasemap/README.md

```diff
@@ -6,18 +6,18 @@ Change a map's basemap. A basemap is beneath all layers on a `Map` and is used t
 
 ## Use case
 
-Basemaps should selected contextually, for example, in maritime applications, it would be more appropriate to use a basemap of the world's oceans as opposed to a basemap of the world's streets.
+Basemaps should be selected contextually, for example, in maritime applications, it would be more appropriate to use a basemap of the world's oceans as opposed to a basemap of the world's streets.
 
 ## How to use the sample
 
-Use the drop down menu to select the active basemap from the list of available basemaps.
+Press the **Change Basemap** button to display the Basemap Gallery. When the gallery appears, select a basemap to be displayed.
 
 ## How it works
 
 1. Create a `Map` object.
 2. Set the map to the `MapView` object.
-3. Choose a new `BasemapStyle` to create a `Basemap` with.
-4. Set the basemap on the map.
+3. Create a `BasemapGallery` using the toolkit.
+4. Use the `setGeoModelFromGeoView` method to manage changes to the basemap.
```

Comment:
I think we should mention the toolkit component somewhere since this is what actually does the logic now

Reply:
we should probably also mention that you will need the toolkit cloned to run this sample yourself